pkg/monitor/cluster/clustersync.go

```diff
@@ -13,57 +13,57 @@ import (
 
 func (mon *Monitor) emitClusterSync(ctx context.Context) error {
 	if mon.hiveClusterManager == nil {
-		// TODO(hive): remove this once we have Hive everywhere.
 		mon.log.Info("skipping: no hive cluster manager")
 		return nil
 	}
 
 	clusterSync, err := mon.hiveClusterManager.GetClusterSync(ctx, mon.doc)
 	if err != nil {
+		mon.log.Errorf("Error in getting the clustersync data %v", err)
```

Comment:
A colon would be nice here to delineate our custom messaging from the underlying error message: 
```suggestion
		mon.log.Errorf("Error in getting the clustersync data: %v", err)
```

Reply:
done